my-app/src/Container/chat/interlocutor/styles/interlocutor.scss

```diff
@@ -0,0 +1,20 @@
+.interlocutor {
+	display: flex;
+	align-items: center;
+	justify-content: space-evenly;
+	width: 100%;
+	height: 20%;
+	background-color: #bacae3;
+
+	.interlocutor__name {
+		background-color: #bacae3;
```

Comment:
looks like too many spaces

Reply:
checked and corrected indentation throughout the document